docs-client/3rd-party-licenses.txt

```diff
@@ -0,0 +1,1618 @@
+THE FOLLOWING SETS FORTH ATTRIBUTION NOTICES FOR THIRD PARTY SOFTWARE THAT MAY BE CONTAINED IN PORTIONS OF THE @ARMERIA/DOCS CLIENT PRODUCT.
```

Comment:
Question: Is this auto-generated? Would it be possible to have this generated into `/licenses`?

Reply:
The file is automatically generated by the command in the README. Do you mean move this file into `armeria/licenses/web-licenses.txt`? Can definitely do that.

Reply:
Yes, please. Could you also update `NOTICE.txt` to point at the `web-licenses.txt` there?